tests/_test_msui/test_mscolab_merge_waypoints.py

```diff
@@ -132,7 +132,9 @@ def test_save_overwrite_to_server(self, mockbox):
         assert wp_server_before.lat != wp_local_before.lat
 
         # trigger save to server action from server options combobox
-        with mock.patch("mslib.msui.mscolab.MscolabMergeWaypointsDialog", AutoClickOverwriteMscolabMergeWaypointsDialog):
+        with mock.patch(
+            "mslib.msui.mscolab.MscolabMergeWaypointsDialog",
+                AutoClickOverwriteMscolabMergeWaypointsDialog):
```

Comment:
I think arguments to the same function should be aligned with the same indentation, i.e. the A should be under the double quote. This is usually done to make them more easily readable at a glance.

Reply:
yea but when i do that i get this error `.\tests\_test_msui\test_mscolab_merge_waypoints.py:137:13: E129 visually indented line with same indent as next logical line  `